app/src/common/shared/org/mozilla/vrbrowser/search/SearchEngineWrapper.java

```diff
@@ -125,6 +125,11 @@ public String getResourceURL() {
         return uri.getScheme() + "://" + uri.getHost();
     }
 
+    public String getHostURL() {
+        Uri uri = Uri.parse(mSearchEngine.buildSearchUrl("")) ;
+        return uri.getHost();
```

Comment:
I understand this is _your_ instrumentation and your data but please consider that all the other products (including Firefox Desktop, Fennec and Fenix) are using the engine `identifier` rather than the host. This doesn't block the review, but you might want to consider using the same.

Reply:
I got `google-b-1-m` from the `SearchEngine.identifier`. Is this what we want? I can follow what they did, no problem.

Reply:
Yes! I think that is exactly what you want. For more context see [the issue](https://github.com/mozilla-mobile/fenix/issues/4082#issuecomment-512187680) that Fenix fixed a bit back.

I'd recommend sticking to what Fenix is doing since they will be improving that part a bit to support mozilla-mobile/fenix#6448